### ISSUE TITLE
[Dungeon] Trial of the Champion 5 hc - The Faceroller

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#5461]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The achievment simply does not work. Tested this last night (09/05/2018). We finished the boss with his hammer and did not get the achievement.
**Expected behaviour**: Tell us what should happen instead
When he uses the hammer on a player that player should get his cast bar switched and given a skill dealing 15k damage. If you kill the boss with this skill you should get the achievement.
**Steps to reproduce the problem**:

1. Leave boss on <15k hp
2. Wait for Hammer of the Righteous to be cast on you and throw it back
3. Notice you not getting the achievement

**Include proofs for this behaviour**

http://www.wowhead.com/achievement=3803/the-faceroller#comments

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3318
